### ISSUE TITLE
NameError: global name 'options' is not defined

### ISSUE THREAD

**Issue by dougroyal** [rethinkdb/rethinkdb#5736]
version: 2.3.0.post1

options is used but never created

```
Traceback (most recent call last):
  File "/usr/local/bin/rethinkdb-import", line 9, in <module>
    load_entry_point('rethinkdb==2.3.0.post1', 'console_scripts', 'rethinkdb-import')()
  File "/usr/local/lib/python2.7/dist-packages/rethinkdb/_import.py", line 929, in main
    import_file(options)
  File "/usr/local/lib/python2.7/dist-packages/rethinkdb/_import.py", line 904, in import_file
    pkey = rdb_call_wrapper(conn_fn, "table check", table_check, db, table, options["create_args"], options["force"])
  File "/usr/local/lib/python2.7/dist-packages/rethinkdb/_backup.py", line 78, in rdb_call_wrapper
    return fn(progress, conn, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/rethinkdb/_import.py", line 885, in table_check
if not options["quiet"]:
NameError: global name 'options' is not defined
```

**Comment by danielmewes** [comment]
Oops this was supposed to be fixed by https://github.com/rethinkdb/rethinkdb/pull/5638 with the 2.3.0.post1 driver update, but we forgot to pick the fix into the right branch before releasing.

I'm going to prepare a new driver release now...

Thanks for bringing this to our attention.

**Comment by danielmewes** [comment]
@dougroyal Can you upgrade to the just released rethinkdb-2.3.0.post2 Python driver and try again? (`pip install --upgrade rethinkdb`)

Sorry for the inconvenience.

**Comment by danielmewes** [comment]
This should be fixed.
